### PR TITLE
Update FD Styles from v0.17 to v0.18

### LINKED ISSUE #2034: Update FD Styles v0.18.0 in Core App

### PR DESCRIPTION
**Pull request by UlianaMunich**
Fixes #2034 

Mainly this ticket is about the new layout for User Menu from Fiori 3. But please check all possible options. User profile with avatar, without, icon instead of avatar, etc.

### CODE REVIEW

**Review by JohannesDoberer — COMMENTED** [review]
Bildschirmfoto 2021-06-09 um 10 47 11
If you hover over the items there should be some padding according to visuals.


Bildschirmfoto 2021-06-09 um 10 59 19
If I have more items in the list in combination with a small display I can't click/see the all entries.

File: core/src/AuthorizationSimpleProfileMenu.html [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+          class="fd-top-nav__icon nav-icon"
+          src="{profileNav.settings.icon}"
+          alt="{profileNav.settings.altText ? profileNav.settings.altText : ''}"
+        />
+        {/if} {/if}
+        <span class="fd-menu__title">{$getTranslation(profileNav.settings.label)}</span>
+      </a>
+    </li>
+    {/if} {#if isAuthorizationEnabled || isProfileLogoutItem} {#if isLoggedIn ||
+    !isAuthorizationEnabled && isProfileLogoutItem}
```
[the remark this answers is not in the document]
Comment (UlianaMunich) [reply core/src/AuthorizationSimpleProfileMenu.html:58]: @JohannesDoberer I guess knows better, haven't implemented that logic so might not know all the pitfalls. @JohannesDoberer do you think we can simplify it? Thanks

[the remark this answers is not in the document]
Comment (JohannesDoberer) [reply core/src/AuthorizationSimpleProfileMenu.html:58]: All is fine we cannot reduce it. Every check is valid.

File: core/src/Authorization.html [hunk 2]
```diff
@@ -1,88 +1,131 @@
...
+          <button
+            class="fd-button fd-button--transparent"
+            tabindex="0"
+            on:click="{onLogoutClick}"
+            data-testid="logout-btn"
+          >
+            {$getTranslation(profileNav.logout.label)}
+          </button>
+        </div>
+        {/if}{/if} {#if isAuthorizationEnabled && !isLoggedIn}
```
Comment (JohannesDoberer) [inline core/src/Authorization.html:115]:
```suggestion
{/if} {#if isAuthorizationEnabled &amp;&amp; !isLoggedIn}        &lt;button          class="fd-button fd-button--transparent"          tabindex="0"          on:click="{startAuthorization}"          data-testid="login-btn"        &gt;          Login        &lt;/button&gt;        {/if}{/if}
```
I would move it inside this block like it was before.

**Comment by JohannesDoberer** [comment]
When I remove the `initials` property in userInfoFn, I'll see there is *undefined* in the avatar of user menu.
I would expect to be empty string/nothing.

**Comment by UlianaMunich** [comment]
@JohannesDoberer good point added an empty string as a fallback.

**Review by JohannesDoberer — APPROVED** [review]

**Pull request merged**